WifiART/src/structs/ArtSystemState.java

```diff
@@ -32,6 +53,99 @@ public ArtSystemState() {
 
     /** Initializes the state variables. */
     public void init() {
-        subelements = new ArrayList<>();
+        lciState = new LciState();
+        zState = new ZState();
+        usageState = new UsageState();
+        bssidState = new BssidState();
+
+        lcrState = new LcrState();
+        mapState = new MapState();
+
+        includedSubelements = new HashMap<>();
+        includedSubelements.put(SubelementName.LCI, false);
+        includedSubelements.put(SubelementName.Z, false);
+        includedSubelements.put(SubelementName.USAGE, false);
+        includedSubelements.put(SubelementName.BSSID, false);
+        includedSubelements.put(SubelementName.LCR, false);
+        includedSubelements.put(SubelementName.MAP, false);
+
+        inputFileName = "";
+        inputDir = "";
+        outputFileName = "";
+        outputDir = "";
+    }
+
+    // Getters for the subelement states
```

Comment:
As tedious as this may seem, getters require javadocs.

Reply:
Yes. Adding Javadoc comments would have further delayed my submission, which is why I didn't include it here. I'm not sure what the standard practice is when it comes to getting feedback on unfinished code, especially with a remote environment. I agree that in the future a depth-first approach would be better than the breadth-first approach I've been pursuing.